README.md

```diff
@@ -6,9 +6,37 @@ Another Validator of OpenAPI spec repository Configuration And Directories.
 
 NPM: https://www.npmjs.com/package/@azure/avocado
 
+## Overview
+
+Avocado validate folder structure and configuration for repository "Azure/azure-rest-api-specs". Avocado report new errors involved in PR. Avocado is a CI tool that will trigger automatically by azure DevOps pipeline when a new pull request is created.

```

Comment:
reports. what's `new errors` mean?  Avocado is not a `CI` tool. it's used by CI ....

Reply:
Updated